package/python-kiwi-spec-template

```diff
@@ -58,6 +58,7 @@ Source:         %{name}.tar.gz
 Source1:        %{name}-rpmlintrc
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildRequires:  gcc
+BuildRequires:  python%{python3_pkgversion} >= 3.6
```

Comment:
Instead of adding this line, just change the line below it to have `>= 3.6` constraint. We require `python%{python3_pkgversion}-%{develsuffix}` at build-time.

Reply:
ok